client/src/app/site/motions/modules/motion-detail/components/motion-detail-diff/motion-detail-diff.component.scss

```diff
@@ -1,15 +1,15 @@
 /* Diffbox */
 .diff-box {
     position: relative;
-    background-color: #f9f9f9;
-    border: solid 1px #eee;
+    // background-color: #f9f9f9;
+    // border: solid 1px #eee;
     border-radius: 3px;
     margin-bottom: 0;
     padding-top: 0;
     padding-right: 40px;
 
     &:hover {
-        background-color: #f0f0f0;
+        // background-color: #f0f0f0;
```

Comment:
the hover effect is missing in motion diff view...

Reply:
Done